### DISCUSSION TITLE
iOS 15.0 SKProductsRequest fails.

### DISCUSSION THREAD

**Discussion by arshad-slate** [hyochan/react-native-iap#1529] [Q&A]
In iOS 15.0 fetching subscription fails with error following error

 ` Background Task 88 ("SKProductsRequest"), was created over 30 seconds ago. In applications running in the background, this creates a risk of termination. Remember to call UIApplication.endBackgroundTask(_:) for your task in a timely manner to avoid this.`

Looks like this is an iOS issue. I haven't checked the latest StoreKit changes in iOS 15. Will do that, but is worth mentioning here.

Thanks.

**Comment by babarmemon-dev** [comment]
getting same issue @arshad-slate did you get the answer?
